ksql-engine/src/test/java/io/confluent/ksql/integration/JsonFormatTest.java

```diff
@@ -187,6 +203,24 @@ public void testSinkProperties() throws Exception {
     assertThat(
         topicClient.describeTopics(ImmutableList.of(streamName)).get(streamName).partitions(),
         hasSize(3));
+    assertThat(topicClient.getTopicCleanupPolicy(streamName), equalTo("delete"));
+  }
+
+  @Test
+  public void testTableSinkCleanupProperty() throws Exception {
+    final String tableName = "SinkCleanupTable".toUpperCase();
+    final int resultPartitionCount = 3;
+    final String queryString = String.format("CREATE TABLE %s AS SELECT * "
+                                             + "FROM %s;",
+                                             tableName, usersTable);
+    executePersistentQuery(queryString);
+
+    TestUtils.waitForCondition(
+        () -> topicClient.isTopicExists(tableName),
+        "Wait for async topic creation"
+    );
+
+    assertThat(topicClient.getTopicCleanupPolicy(tableName), equalTo("compact"));
```

Comment:
here and elsewhere we should be checking equivalence to `TopicCleanupPolicy.COMPACT` etc.